src/PropertyAccessor/PropertyReaderInterface.php

```diff
@@ -10,14 +10,14 @@
 interface PropertyReaderInterface
 {
     /**
-     * @param $object
+     * @param object|array $object
```

Comment:
This was added to be more correct, but the `ArrayPropertyReader` breaks the Liskov principle... What can we do?